web/index.ts

```diff
@@ -355,6 +355,20 @@ const App = (_: any, state: AppState, setState: SetState) => {
                                     setLoadingState({ heights: clone });
                                 }
                             })
+                        ),
+                        H('div',
+                            { className: 'field-flex' },
+                            H(Button, {
+                                label: `Remove Image ${i + 2}`,
+                                onclick: (e: MouseEvent) => {
+                                    e.preventDefault();
+                                    const filter = (arr: any[]) => [...arr].filter((_, n) => n !== i + 1);
```

Comment:
```suggestion
                                    const filter = (arr: string[]) => arr.filter((_, n) => n !== i + 1);
```

Reply:
Happy to correct the type.

The spreading of the array is needed however; if for example a user only sets the width/height of the 4th image it will make those arrays defined as `[4: 350]`. The suggested method leads to the width/height arrays being filtered incorrectly, as the leading `undefined`s are not created .

See https://codepen.io/spinks_/pen/xxVdPBN for how the two filters apply would apply in this case.

At least this is what I found testing it, doing it with the suggested method leads to the widths/heights being moved around incorrectly due to that behaviour.

Reply:
Nice catch, sounds good 👍